layers/relay/Dockerfile

```diff
@@ -1,12 +1,18 @@
 # syntax = docker/dockerfile:1.4
+ARG CPU_PREFIX
 ARG PHP_VERSION
 ARG BREF_VERSION
-FROM bref/build-php-$PHP_VERSION:2 AS ext
+FROM bref/${CPU_PREFIX}build-php-$PHP_VERSION:$BREF_VERSION AS ext
 
 ARG RELAY_VERSION=0.6.0
 
+ENV LD_LIBRARY_PATH=/usr/lib:/usr/lib64:$LD_LIBRARY_PATH
+
 # Docs: https://relay.so/docs/1.x/installation#manual-installation
 
+# Added openssl11-libs
```

Comment:
Not sure this is a good idea. We don't want openssl stuff of a different version to in the main image?

Reply:
The copy-dependencies script was failing due to missing library. Was not sure how to fix it. 

> [ext 5/7] RUN php /bref/lib-copy/copy-dependencies.php /tmp/relay.so /tmp/extension-libs:
#14 0.507
#14 0.507 Fatal error: Uncaught RuntimeException: This library is a dependency for /tmp/relay.so but cannot be found by 'ldd':
#14 0.507 	libssl.so.1.1 => not found

>bash-4.2# ldd /tmp/relay.so
ldd: warning: you do not have execution permission for `/tmp/relay.so'
	linux-vdso.so.1 (0x0000ffff96b83000)
	libpthread.so.0 => /usr/lib64/libpthread.so.0 (0x0000ffff96a57000)
	libssl.so.1.1 => not found
	libcrypto.so.1.1 => not found
	libzstd.so.1 => /opt/lib/libzstd.so.1 (0x0000ffff96956000)
	liblz4.so.1 => /usr/lib64/liblz4.so.1 (0x0000ffff96925000)
	libanl.so.1 => /usr/lib64/libanl.so.1 (0x0000ffff96904000)
	libc.so.6 => /usr/lib64/libc.so.6 (0x0000ffff9677e000)
	/lib/ld-linux-aarch64.so.1 (0x0000ffff96b55000)

Reply:
or we could raise an issue upstream with them?

Reply:
Removing is also a good option. If someone needs it they can always do a PR. My main focus was to make the whole bunch arm compatible and I don't use any of the extension except for GD and imagick. 

cassandra and sqlsrv doesn't have arm support

Reply:
Probably we should make a separate PR to remove/fix any broken extensions we discover?

Reply:
👌